.github/workflows/main.yaml

```diff
@@ -8,13 +8,13 @@ jobs:
       contents: write
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4.2.2
```

Comment:
@BernardoScantamburlo you can just do the major version, like checkout@4 and it will automatically upgrade us for the minor and patch versions. Btw, semantic version is major.minor.patch see https://semver.org